test/invariant/stores/TimestampStore.sol

```diff
@@ -11,7 +11,7 @@ contract TimestampStore {
         currentTimestamp = uint32(block.timestamp);
     }
 
-    function increaseCurrentTimestamp(uint32 timeJump_) external {
-        currentTimestamp += timeJump_;
+    function increaseCurrentTimestamp(uint32 timeJump_) external returns (uint32) {
+        return currentTimestamp += timeJump_;
```

Comment:
looks suspicious, do we ever increase `currentTimestamp` in this store? 
If no, is this contract needed?

How timestamp is stored between multiple calls?

Reply:
Yes we do and we use the stored value in the `useCurrentTimestamp` modifier:
https://github.com/MZero-Labs/protocol/blob/ad9ab6c53f5e0807c8df4a303e50476f69e6b91c/test/invariant/Invariant.t.sol?plain=1#L36

And also when checking the rate confidence interval:
https://github.com/MZero-Labs/protocol/blob/ad9ab6c53f5e0807c8df4a303e50476f69e6b91c/test/invariant/handlers/ProtocolHandler.sol?plain=1#L48

Reply:
I saw it as `return currentTimestamp + timeJump_`